### PR TITLE
Profile Admin/General User

### PR DESCRIPTION
**Pull request by whatzerrname**
Design Profile Admin/General User

### CODE REVIEW

**Comment by whatzerrname** [comment]
Design Profile Admin/General User

**Comment by whatzerrname** [comment]
[Admin - User profil](https://trello.com/c/m2o0T9Lr/14-admin-user-profil)

File: .idea/copyright/profiles_settings.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```
Comment (qdequippe) [inline .idea/copyright/profiles_settings.xml]: Add it to .gitignore

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (qdequippe) [inline .idea/misc.xml]: Add it to .gitignore

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (qdequippe) [inline .idea/modules.xml]: Add it to .gitignore

File: .idea/sqldashboard.iml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (qdequippe) [inline .idea/sqldashboard.iml]: Add .idea folder to .gitignore

File: src/AppBundle/Resources/views/App/pagination.html.twig [hunk 5]
```diff
@@ -0,0 +1,61 @@
+{#
```
Comment (qdequippe) [inline src/AppBundle/Resources/views/App/pagination.html.twig:1]: Think about https://github.com/KnpLabs/KnpPaginatorBundle maybe later

File: src/AppBundle/Resources/views/App/userProfile.html.twig [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    <html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        {% block stylesheets %}
+            {{ parent() }}
+            <link href="{{ asset('assets/css/style.css') }}" rel="stylesheet" />
+        {% endblock %}
+        <title>{% block title %}Profile Admin{% endblock %}</title>
```
Comment (qdequippe) [inline src/AppBundle/Resources/views/App/userProfile.html.twig]: why do you need to include all headers to update the title and/or css check https://github.com/Studeal/sqldashboard/pull/2/files

File: web/assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
+* {
```
Comment (qdequippe) [inline web/assets/css/style.css]: maybe put this in user.css

**Review by qdequippe — CHANGES_REQUESTED** [review]

File: src/AppBundle/Resources/views/App/userProfile.html.twig (hunk 6 above)
Comment (whatzerrname) [reply src/AppBundle/Resources/views/App/userProfile.html.twig]: Bien vu 🙈

**Review by whatzerrname — COMMENTED** [review]
Review:
- .idea added to .gitignore
- OK for deletion of index.html.twig
- KNP Pagination on to do list
- OK to remove redundant stylesheet links on UserProfile
- Created user.css

File: .idea/copyright/profiles_settings.xml (hunk 1 above)
Comment (qdequippe) [reply .idea/copyright/profiles_settings.xml]: @whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file

File: .idea/misc.xml (hunk 2 above)
Comment (qdequippe) [reply .idea/misc.xml]: @whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file

File: .idea/modules.xml (hunk 3 above)
Comment (qdequippe) [reply .idea/modules.xml]: @whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file

File: .idea/sqldashboard.iml (hunk 4 above)
Comment (qdequippe) [reply .idea/sqldashboard.iml]: @whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file

File: .idea/copyright/profiles_settings.xml (hunk 1 above)
Comment (whatzerrname) [reply .idea/copyright/profiles_settings.xml]: Est-ce que c'est bon maintenant? @qdequippe

**Review by qdequippe — CHANGES_REQUESTED** [review]
.idea folder is still present. You need to remove it

**Review by qdequippe — APPROVED** [review]

**Pull request merged**